internal/services/containers/kubernetes_cluster_network_resource_test.go

```diff
@@ -1343,6 +1372,85 @@ resource "azurerm_kubernetes_cluster" "test" {
 `, data.RandomInteger, data.Locations.Primary, internalIngressEnabled, externalIngressEnabled)
 }
 
+func (KubernetesClusterResource) serviceMeshProfileWithRevisions(data acceptance.TestData, internalIngressEnabled bool, externalIngressEnabled bool) string {
```

Comment:
This is touching add-on functionality so can we move this to `kubernetes_cluster_addons_resource_test.go` file and use the test naming convention in that file

Reply:
Updated in latest commit